### DISCUSSION TITLE
MessagesSearch always returns `messages` as nil

### DISCUSSION THREAD

**Discussion by pyxelpioneer** [gotd/td#1788] [Q&A]
### What version of gotd are you using?

github.com/gotd/td v0.99.1

### Can this issue be reproduced with the latest version?

Yes

### What did you do?

I am trying to get all the pinned messages in a peer.

Here is the code:

```go
pinnedMessages, err := Client.API().MessagesSearch(ctx, &tg.MessagesSearchRequest{
            Peer: utils.PeerAsInputPeer(peer, accessHash),
            Filter: &tg.InputMessagesFilterPinned{},
})

if err != nil {
        log.Fatalf("Error: %v", err)
}
```

### What did you expect to see?

I was expecting a list of messages that represents all the messages

### What did you see instead?

It is null and on printing (json-ing) the value of `pinnedMessages`, i am getting:

```json
{"Flags":0,"Inexact":false,"Count":1,"NextRate":0,"OffsetIDOffset":0,"Messages":null,"Chats":null,"Users":null}
```

I can confirm that I am getting correct amount of `Count` but why are the `Messages` null? Am i missing smth?

### What Go version and environment are you using?

go version go1.22.0 linux/amd64

**Comment by ernado** [comment]
I've transfered this issue to discussins since it is Telegram Server behaviour.
